src/Date/DatePickerModalHeader.tsx

```diff
@@ -32,7 +33,11 @@ export default function DatePickerModalHeader(
               color={color}
             />
             <Appbar.Content title={''} />
-            <Button color={color} onPress={props.onSave}>
+            <Button
+              color={color}
+              onPress={props.onSave}
+              disabled={!props.state.date}
```

Comment:
I don't want to disable buttons, I think the error should be sufficient enough

Reply:
Backup: https://uxdesign.cc/why-you-shouldnt-include-disabled-interaction-elements-in-your-design-system-76a2d4307faf